prov/gni/test/sep.c

```diff
@@ -222,6 +222,11 @@ void sep_setup(void)
 			ret = fi_enable(tx_ep[i][j]);
 			cr_assert(!ret, "fi_enable");
 
+			/* test if bind fails */
+			ret = fi_ep_bind(tx_ep[i][j], &tx_cq[i][j]->fid,
+					 FI_TRANSMIT);
+			cr_assert(ret, "fi_ep_bind should fail");
```

Comment:
Same comment I gave to Howard in his PR.  This should be a separate test, not in every test.